### ISSUE TITLE
Why rescale data?

### ISSUE THREAD

**Issue by Bisaloo** [AckerDWM/gg3D#1]
Hi,

Thank you for this package! I may be missing something obvious but why rescale data? It makes it impossible to do some semi-complex operations such as:

``` r
library(ggplot2)
library(gg3D)

data(iris)
iris2 <- iris
iris2$Petal.Width <- iris2$Petal.Width + 10

g <- ggplot(iris, aes(x=Petal.Width, y=Sepal.Width, z=Petal.Length, color=Species)) + 
  theme_void() +
  axes_3D() +
  stat_3D()

g2 <- g +
  stat_3D(aes(x=Petal.Width, y=Sepal.Width, z=Petal.Length, color=Species), iris2)
```

Here, we expect `g` and `g2` to be different since `g2` has an additional layer with different data but they look just the same.

**Comment by AckerDWM** [comment]
That's a good point. I usually only use 3D for quickly visualizations, hence scaling minimizes how much work I have to do prior to plotting.

I will probably make scaling optional in an update. For now, a workaround would be to combine iris and iris2 into a single dataframe.

**Comment by Bisaloo** [comment]
> For now, a workaround would be to combine iris and iris2 into a single dataframe.

Good point. I guess my example was not so good. I actually want to use different `geom` for my two datasets. This is why I am using two datasets.

**Comment by Bisaloo** [comment]
Would you like me to submit a PR to try and fix this?

**Comment by AckerDWM** [comment]
Please do; happy to have you work on it.

**Comment by Bisaloo** [comment]
It's harder than what I anticipated (but you probably already knew that :smiley:). 

`perspbox` output will depend on the limits for `x`, `y` and `z`. Those limits can change between the different `stat_3D` layers. I think we would need a way to somehow exchange information between layers so limits in `perspbox` can be set for *all* layers. I'm not sure it's possible...

What do you think? Maybe you already gave it some thought and have more ideas than me.

**Comment by AckerDWM** [comment]
You could add limit parameters, xlim, ylim, zlim, specified in the stat_3D or aes call. These would add dummy points for the perspective transformation. It might also be possible in the more flexible geom system, but I haven’t bothered to learn that.
